ansible/ansible_collections/agentic/agent_configs/tests/unit/plugins/modules/test_agent_config_modules.py

```
@@ -684,12 +991,13 @@ def test_json_file_reports_write_failures(
     tmp_path: Path, monkeypatch: pytest.MonkeyPatch
 ) -> None:
     """Verify json_file surfaces an atomic-write failure through fail_json."""
+
     def fail_write(path: str, content: str) -> None:
```

Ruff (0.15.12)

[warning] Unused function argument: `path`

(ARG001)